tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_test.ts

```diff
@@ -401,5 +408,52 @@ describe('linked_time_fob_controller', () => {
         end: {step: 3},
       });
     });
+
+    it('changing start input modifies start step', () => {
+      let fixture = createComponent({
```

Comment:
Replace 'let' with 'const' wherever it is possible in this file.

Reply:
Done.